.gitignore

```diff
@@ -1,23 +1,2 @@
-# Python
 __pycache__/
 *.py[cod]
-*.pyo
-*.pyd
-*.env
-*.venv
-env/
-venv/
-ENV/
-VENV/
-*.egg-info/
-dist/
-build/
-
-# Node.js
-node_modules/
```

Comment:
we cant have this versioned 😭

Reply:
Yes, Git LFS (Large File Storage) introduces a new development dependency. Developers working with repositories that use Git LFS need to have it installed locally to handle large files properly. Git LFS replaces large files in your repository with text pointers, and the actual files are stored separately. Without Git LFS installed, these files won't be accessible, and operations like cloning or pulling the repository may not work as expected.

Migrating Git LFS files from GitHub to a self-hosted GitLab instance requires careful planning:

- Install and configure Git LFS on your GitLab server.
- Migrate LFS objects using tools like git lfs migrate or by pulling the data from GitHub and pushing it to GitLab.
- Ensure all team members install Git LFS locally and update their remote URLs.
- Monitor storage and bandwidth usage on your server.
- Update any integrations or services that interact with your repositories.